remoting/getty/config.go

```diff
@@ -177,6 +182,23 @@ func (c *GettySessionParam) CheckValidity() error {
 	return nil
 }
 
+func parseTimeDurationByRange(timeStr string, min *time.Duration, max *time.Duration) (time.Duration, error) {
```

Comment:
What if the value of `min` or  the `max` is less than 0? The boundary conditions should be fully considered.